functions/index.js

```diff
@@ -0,0 +1,25 @@
+process.env.DEBUG = 'nuxt:*';
+
+const functions = require('firebase-functions');
+const { loadNuxt } = require('nuxt');
+
+let isReady = false;
+
+let nuxt = loadNuxt('start');
+
+async function handleRequest(req, res) {
+  try {
+    if (!isReady) {
+      nuxt = await nuxt;
+      isReady = true;
+    }
+    console.log(req.path);
+    res.setHeader('Cache-Control', 'public, max-age=3600, s-maxage=7200');
```

Comment:
the examples I see all use `res.set` not `res.setHeader` - do they do the same thing?